### ISSUE TITLE
Make announceStopViaConsole configurable

### ISSUE THREAD

**Issue by ChipWolf** [itzg/mc-server-runner#13]
The proposed enhancement here is to read a list of commands, perhaps from a file at the root of the server to execute a custom set of shutdown commands

The current implementation uses a single 'say' command, our use-case is a preference for 'title' commands at intervals before a shutdown.

c.c. @morrissimo

**Comment by morrissimo** [comment]
Definitely like the idea - already thought of having the last 5 seconds countdown to shutdown, like I've seen on other servers. 

But: I would need guidance from @itzg on this one, as I've got no idea how to do something like run an arbitrary script given the current Go harness

**Comment by itzg** [comment]
Yeah, was also thinking that idea/request would show up soon 😀 I don't have any particular guidance or hesitation with reading a script file, other than I think leveraging the builtin Go templating would be cool for populating variables as needed, such as time to shutdown.

**Comment by ChipWolf** [comment]
I'd probably do something as simple as running a shell script, a bunch of `rcon-cli` commands with variable substitutions maybe?

**Comment by jemand771** [comment]
just as a minor addition (as it somehow relates to this feature request):
kicking all users with a (possibly configurable) message before stopping the server could also improve the experience for players.
e.g. when the server restarts while someone is connected and afk, they'll see the "hey, the server is restarting" message instead of "server closed"
